alacritty/src/display/cursor.rs

```diff
@@ -24,6 +24,7 @@ impl IntoRects for RenderableCursor {
 
         if self.is_wide() {
             width *= 2.;
+            thickness /= 2.;
```

Comment:
Why is the thickness doubled here in the first place? Looks like you're fixing a symptom here.

Reply:
I see what you mean.